### DISCUSSION TITLE
arduino 2.3.2

### DISCUSSION THREAD

**Discussion by PGZyeKOcLkOno8vc** [PE5PVB/TEF6686_ESP32#627] [Q&A]
Nice to meet you, I added three types of libraries as specified in arduino 2.3.2 and modified User_Setup.h as well.

I built source V2.10

'class TFT_eSPI' has no member named 'setSPISpeed'

I get an error and can't build, what could be the cause?

At 1.61, there was no problem at all.
For some reason, there is no problem with platformIO, but I want to use arduino.

**Comment by PE5PVB** [comment]
Please use the modified tft_espi library: https://github.com/PE5PVB/TFT_eSPI_DynamicSpeed

**Reply by PGZyeKOcLkOno8vc** [reply]
Thanks for the quick reply.

The library was old from V1.16.

Replaced the pointed library.

The TFT_espi-related error disappeared, but the following error appeared.

Compilation error: 'gpio_set_drive_capability' was not declared in this scope; did you mean 'gpio_ll_set_drive_capability'?

So, there was no problem before https://github.com/espressif/arduino-esp32

I dropped the version from 3.0.3 to 2.0.17 and it was OK.

Apparently it is not good to make it 3.0.0 or higher.

**Comment by PE5PVB** [comment]
Please downgrade the ESP32 boardmanager to the previous version (< v3.0.x)

**Reply by PGZyeKOcLkOno8vc** [reply]
Downgrade OK. All you have to do is move. Thank you very much. I'm having fun with my own printed circuit board. We look forward to working with you in the future.
